packages/node/src/evm/initialization.ts

```diff
@@ -1,221 +0,0 @@
-import { ethers } from 'ethers';
```

Comment:
Removing this is a significant improvement to the code 🎉

Reply:
I have to admit that it was super scary to remove all of these and had to read that code quite a few times before actually decide on getting rid of it but this will make the setup process of an airnode much more simple than it is today.